tf/tfprocess.py

```diff
@@ -258,12 +267,6 @@ def process(self, batch_size, test_batches, batch_splits=1):
         # Update steps since training should have incremented it.
         steps = tf.train.global_step(self.session, self.global_step)
 
-        # Determine learning rate
-        lr_values = self.cfg['training']['lr_values']
-        lr_boundaries = self.cfg['training']['lr_boundaries']
-        steps_total = (steps-1) % self.cfg['training']['total_steps']
```

Comment:
because this is after the step increment as training for the step has been run.  Thus global step counter may actually be a multiple of total_steps - and reporting that the LR just used is the first LR rather than the last LR would be pretty obviously wrong.